puma/hlplots/aux_results.py

```diff
@@ -1,32 +1,70 @@
 """Auxiliary task results module for high level API."""
 from __future__ import annotations
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from pathlib import Path
 
 import numpy as np
-from ftag import Cuts
+from ftag import Cuts, Flavour, Flavours
 from ftag.hdf5 import H5Reader
 
-from puma.hlplots.results import Results
 from puma.hlplots.tagger import Tagger
+from puma.utils import logger
 from puma.utils.vertexing import calculate_vertex_metrics
 from puma.var_vs_aux import VarVsAux, VarVsAuxPlot
 
 
+def get_aux_labels():
```

Comment:
Might be best to move this information upstream somewhere (maybe FTAG tools?), but for now I just left it here.

Reply:
Sounds good, maybe we can create an issue in FTAG tools and deal with it later if we decide it would be useful.